build.gradle

```diff
@@ -53,61 +44,42 @@ allprojects {
                 password System.getenv("packagePAT")
             }
         }
-
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
     }
 
     ext {
-        snapshotVersion= '-SNAPSHOT'
+        snapshotVersion = '-SNAPSHOT'
         timestampedVersionRegex = '.*-\\d{8}-\\d{6}-\\w.*\$'
     }
 }
-configurations {
-    balTools
-}
 
-dependencies {
-    balTools ("org.ballerinalang:jballerina-tools:${ballerinaLangVersion}") {
-        transitive = false
+subprojects {
+    configurations {
+        ballerinaStdLibs
+        jbalTools
     }
+
+    dependencies {
+        /* JBallerina Tools */
+        jbalTools("org.ballerinalang:jballerina-tools:${ballerinaLangVersion}") {
```

Comment:
Ideally these should not based on the intermediate pack but complete distribution.